pack/tic/lod_encounter.json

```diff
@@ -0,0 +1,546 @@
+        "clues": 1,
+        "code": "07289",
+        "double_sided": true,
+        "encounter_code": "the_lair_of_dagon",
+        "encounter_position": 16,
+        "faction_code": "mythos",
+        "illustrator": "Ed Mattinian",
+        "name": "Hall of Silence",
+        "pack_code": "lod",
+        "position": 289,
+        "quantity": 1,
+        "shroud": 4,
+        "text": "<b>Forced</b> - After Hall of Silence is revealed: Place the set-aside purple key on it.\n<b>Forced</b> - After an investigator at this location fails a skill test: Add 1 [curse] token to the chaos bag.",
+        "traits": "Third Floor.",
+        "type_code": "location"
+    },
+    {
+        "back_flavor": "The air is frigid here, and there is no sound but the faint plinking of water dripping onto the rough stone ground. You shudder and proceed onward...",
+        "back_illustrator": "Patrick McEvoy",
+        "back_name": "Tidal Tunnel",
+        "clues": 1,
+        "clues_fixed": true,
+        "code": "07290",
+        "double_sided": true,
+        "encounter_code": "the_lair_of_dagon",
+        "encounter_position": 17,
+        "faction_code": "mythos",
+        "illustrator": "Greg Bobrowski",
+        "name": "Doorway to the Depths",
+        "pack_code": "lod",
+        "position": 290,
+        "quantity": 1,
+        "shroud": 5,
+        "text": "<Forced</b> - After Doorway to the Depths is revealed: Place the set-aside green key on the location with the most clues.\n[action] Investigators at the Doorway to the Depths spend the green key and 3 [per_investigator] clues, as a group: Remember that you have \"unlocked the final depths\".",
+        "traits": "Cave.",
+        "type_code": "location"
+    },
+    {
+        "back_flavor": "The steep, twisting stairway beyond the doorway leads you deeper and deeper into the caverns below the building. Effigies depicting aquatic monstrosities watch you from both sides as you descend into a realm of alien architecture and otherworldly riches.",
+        "clues": 3,
+        "code": "07291",
+        "double_sided": true,
+        "encounter_code": "the_lair_of_dagon",
+        "encounter_position": 18,
+        "faction_code": "mythos",
+        "illustrator": "Daria Khlebnikova",
+        "name": "Lair of Dagon",
+        "pack_code": "lod",
+        "position": 291,
+        "quantity": 1,
+        "shroud": 3,
+        "text": "<b>Forced</b> - After you enter Lair of Dagon: Test your lowest skill (3). If you fail, take 1 horror.\n[action]: Test [willpower] (3) or [agility] (3). If you succeed, place 1 of your clues on the current act. You may spend 1 key to place 1 more of your clues on the current act.",
+        "traits": "Y'ha-nthlei. Lair.",
+        "type_code": "location",
+        "victory": 1
+    },
+	{
+		"code": "07292",
+		"encounter_code": "the_lair_of_dagon",
+		"encounter_position": 19,
+		"enemy_evade": null,
+		"enemy_fight": null,
+		"faction_code": "mythos",
+		"health": null,
```

Comment:
Believe this one should be `"hidden": true` since its a back side.  Also perhaps it should be `07292a` like the card